src/commands/settings/settings-karma.ts

```diff
@@ -0,0 +1,34 @@
+import type { ChatInputCommandInteraction } from 'discord.js';
+
+import { getChannelFirestoreReference } from '../../utilities/firestore-helper.js';
+
+/** Enables or disables karma tracking on a channel in Firebase */
+export async function karma(
```

Comment:
Not the biggest fan of this file naming convention. Instead could we maybe do folder structure? something `settings/karma/index.ts` or even `settings/karma/karma.ts`

It just feels like this could be clunky if we ever have multiple nested sub commands, and there is a bit of self evidence already with the command being in the settings folder.

Reply:
In the next PR it is renamed to config so we can plan any future edits for beyond that